docs/topics/request-response.rst

```diff
@@ -693,9 +693,22 @@ Response objects
     :param ip_address: The IP address of the server from which the Response originated.
     :type ip_address: :class:`ipaddress.IPv4Address` or :class:`ipaddress.IPv6Address`
 
+    :param protocol: A tuple containing information about the protocol that was used
+        to download the response. Taken from the ``version`` attribute of the
+        corresponding :class:`twisted.web.client.Response` object, it will tipically
+        consist of the protocol and version numbers, e.g. ``(b"HTTP", 1, 1)``
+        to represent "HTTP/1.1".
+    :type protocol: :class:`tuple`
+
+    .. versionadded:: 2.0.0
+       The ``certificate`` parameter.
+
     .. versionadded:: 2.1.0
        The ``ip_address`` parameter.
 
+    .. versionadded:: 2.X.X
```

Comment:
I think @Gallaecio suggested using VERSION, is this right?

Reply:
Do you mean as a placeholder, to have a standardized string to search for and replace before releasing?